src/LoRa.h

```diff
@@ -54,6 +54,8 @@ class LoRaClass : public Stream {
   void setSyncWord(int sw);
   void enableCrc();
   void disableCrc();
+  // Over Current Protection control (Semtech SX1276/77/78/79 5.4.4.)
+  void setOCP_sx127x(uint8_t mA);
```

Comment:
I don't think this needs to be a public API anymore. Maybe the name can also be changed to `setOCP` as well?